### PR TITLE
CHECK_TICK

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

File: code/game/gamemodes/setupgame.dm [hunk 1]
```diff
@@ -14,6 +15,7 @@
 			bad_mutations |= B
 		else if(B.quality == MINOR_NEGATIVE)
 			not_good_mutations |= B
+		CHECK_TICK
 
 /datum/subsystem/ticker/proc/setupFactions()
```
Comment (KazeEspada) [inline code/game/gamemodes/setupgame.dm]: THIS DOESNT EVEN DO ANYTHING

Comment (Cheridan) [inline code/game/gamemodes/setupgame.dm]: >kaze doesn't know

Comment (KazeEspada) [inline code/game/gamemodes/setupgame.dm]: faction system?

**Comment by optimumtact** [comment]
Is any of this code even setup for sleeping in the middle of the loop?

Have you ensured none of it is relying on the loop finishing in one given process run?

File: code/game/gamemodes/setupgame.dm (hunk 1 above)
Comment (JJRcop) [inline code/game/gamemodes/setupgame.dm]: #n1ic6e g0ue8s9s

**Comment by MrStonedOne** [comment]
I have, and ya, it should be fine.

These all run only from initialize, not fire

**Pull request merged**